### DISCUSSION TITLE
无法升级年度会员？

### DISCUSSION THREAD

**Discussion by IntecL** [immersive-translate/immersive-translate#2262] [Help] [answered]
### 概述 | Summary

一点击升级按钮，就报错
![image](https://github.com/user-attachments/assets/26326abc-18e3-438e-93eb-0656b0e014e0)

**Comment by leonardleonard** [comment, marked as answer]
请提供下邮箱或者发下信息给 support@immersivetranslate.com

**Reply by IntecL** [reply]
今天又试了下，可以升级了，可能是网络问题吧。我这边把问题关闭了
